src/main/java/org/example/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,17 @@
+package org.example.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
We need to use Getter and Setter instead of data for JPA Entity, but it's ok to just use Data for DTOs